docs/roadmap.md

```diff
@@ -0,0 +1,27 @@
+# Roadmap
+
+The list is based on the survey results and the discussion in wg-etcd-operator. It is not set in stone and may be adjusted as needed.
+
+## v0.1.0
+- Create a new etcd cluster, e.g 3 or 5 members cluster
+  - Users should be able to set at least the etcd version and cluster size
+- Understand health of a cluster
+- Enabling TLS communication
+  - Should also support certificate renewal
+
+## v0.2.0
+- Upgrade across patches or one minor version
+- Scale in and out, e.g 3 members to 5 members or vise versa
```

Comment:
I think there is no value in supporting > 5 members, right?
```suggestion
- Scale in and out, 1 -> 3 -> 5 members and vise versa
```

Reply:
resolved & thx